example/case_study.jl

```diff
@@ -123,20 +130,27 @@ n = 100
 init_params = (
     μ_BSLD=60,μ_d=0.0055, μ_g=0.0015, μ_ϕ=0.2,
     ω_BSLD=0.7 ,ω_d=1.0, ω_g=1.0 , ω_ϕ=1.5, σ=0.18,
-    κ = 1, λ=1450, β = 0,
+    λ=1450, β = 0,
     η_BSLD=zeros(n) ,η_d=zeros(n), η_g=zeros(n) , η_ϕ=zeros(n),
     )
 
 
 identity_link_model = identity_link(longit_id, longit_time, longit_sld, surv_id, surv_time, surv_indicator)
-identity_link_chn = sample(identity_link_model, NUTS(100, 0.9, max_depth = 8), 200, init_params=init_params)
-
-# [κ, λ, β] = [1.1500626704863541, 3.9408207824225534, 0.026148519352791527]
+identity_link_chn = sample(identity_link_model, NUTS(50, 0.9, max_depth = 8), 100, init_params=init_params)
```

Comment:
```suggestion
identity_link_chain = sample(identity_link_model, NUTS(50, 0.9, max_depth = 8), 100, init_params=init_params)
```
since the name is already long, and these 2 additional characters simplify the reading